### PR TITLE
Add nanovar

### PR DESCRIPTION
**Pull request by cytham**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by cytham** [comment]
@bioconda/core

Requesting PR to be reviewed and merged.

Thanks!
cytham

File: recipes/nanovar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - biopython ==1.74
+    - keras ==2.2.4
+    - matplotlib ==2.2.3
+    - natsort ==6.2.0
+    - numpy ==1.17.3
+    - progress ==1.4
+    - pybedtools ==0.8.0
+    - python >=3.5
+    - scipy ==1.2.1
+    - tensorflow ==1.13.1
```
Comment (dpryan79) [inline recipes/nanovar/meta.yaml]: Do we really need to ping to these exact versions, rather than these being minimum versions?

Comment (cytham) [reply recipes/nanovar/meta.yaml]: I encountered conflicts/error when I do that. Let me try again now.

Comment (cytham) [reply recipes/nanovar/meta.yaml]: I have resolved the error. Version 1.2.2 allows those dependencies being minimum versions. Should make a new commit?

Comment (dpryan79) [reply recipes/nanovar/meta.yaml]: Yes please!

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
I'll try merging this with the bot after lunch, since I need to keep testing that.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by cytham** [comment]
thanks!
